config/hsc/fgcmBuildStarsTable.py

```diff
@@ -0,0 +1,29 @@
+"""
+HSC-specific overrides for FgcmBuildStarsTable
+"""
+
+import os.path
+
+
+config.minPerBand = 2
+config.densityCutMaxPerPixel = 2000
+config.filterMap = {'g': 'g', 'r': 'r', 'r2': 'r', 'i': 'i', 'i2': 'i',
+                    'z': 'z', 'y': 'y',
+                    'N387': 'N387', 'N816': 'N816', 'N921': 'N921',
+                    'N1010': 'N1010'}
+config.primaryBands = ('i', 'r', 'g', 'z', 'y', 'N387', 'N816', 'N921', 'N1010')
+config.doSubtractLocalBackground = True
+
+config.fgcmLoadReferenceCatalog.refObjLoader.ref_dataset_name = 'ps1_pv3_3pi_20170110'
+config.fgcmLoadReferenceCatalog.refFilterMap = {'g': 'g', 'r': 'r', 'r2': 'r',
```

Comment:
I was confused by this line and the above `filterMap` because I believe our LoadReferenceObjectsTask does not need matching filter names to be defined in the `filterMap` (e.g. how our PS1 refcats work, because most filter names exist in both). Maybe add a comment to that effect, because this is not the same thing?